packages/manager/apps/dedicated/client/app/components/dedicated-cloud/service-pack/upgrade/upgrade.html

```diff
@@ -1,7 +1,14 @@
 <oui-back-button data-on-click=":: $ctrl.goBack()">
     <span data-ng-bind=":: $ctrl.backButtonText"></span>
 </oui-back-button>
-
+<oui-message
+    data-ng-if="$ctrl.isPremier && $ctrl.nsxStatus !== 'enabled'"
```

Comment:
Could we rely on a function call/instance check instead of harcoding this kind of condition in the view ?